src/components/services/CopyToClipboard.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <Generic :item="item">
+    <template #indicator>
+      <div class="status"><i ref="copyIcon" class="fa-regular fa-copy fa-xl" :class="{'scale': animate}" @click="copy()"></i></div>
+    </template>
+  </Generic>
+</template>
+
+<script>
+import service from "@/mixins/service.js";
+import Generic from "./Generic.vue";
+
+export default {
+  name: "CopyToClipboard",
+  mixins: [service],
+  props: {
+    item: Object,
+  },
+  components: {
+    Generic,
+  },
+  data: () => ({
+    animate: false
+  }),
+  mounted: function() {
+    this.$refs.copyIcon.addEventListener(
+      'animationend',
+      () => { this.animate = false; });
+  },
+  methods: {
+    copy() {
+      javascript:navigator.clipboard.writeText(this.item.clipboard)

```

Comment:
Is there a reason to use the `javascript:` prefix? I think this should be fine:  

```suggestion
      navigator.clipboard.writeText(this.item.clipboard)
```

Reply:
True, there's no need to use the `javascript:` prefix. I just took the code from the example in the issue. Removed.